functions/tree_calculation.py

```diff
@@ -0,0 +1,22 @@
+import colorama
+from colorama import Fore

```

Comment:
This is already imported when you imported colorama in line 1.

Reply:
I just tried to remove it, but got error `NameError: name 'colorama' is not defined`, wouldn't work without either of them unfortunately :'(